tests/system/test_requests.py

```diff
@@ -1,20 +1,21 @@
+from collections import defaultdict
+import gzip
+import json
+import requests
+import threading
+import time
+import zlib
+
 from nose.tools import raises
+from requests.exceptions import SSLError
 
 from apmserver import ServerBaseTest, SecureServerBaseTest, ClientSideBaseTest, CorsBaseTest
-from requests.exceptions import SSLError
-import requests
-import json
-import zlib
-import gzip
-import time
-from datetime import datetime
-from collections import defaultdict
-import threading
+
 
 try:
     from StringIO import StringIO
 except ImportError:
-    import io
+    from io import StringIO
```

Comment:
shouldn't this work for both python2.7 in python3?

Reply:
yup, as long as you only feed unicode strings into `io.StringIO`, it should behave the same as `StringIO.StringIO` on 2.7. The latter also accepts bytestrings